i18n/en/aspectRatio.json

```diff
@@ -1,16 +1,16 @@
 {
-  "ratioPreview": "Ratio Preview",
```

Comment:
This one should be English, I think we got this file swapped with Swahili?

Reply:
Sure lemme work on it